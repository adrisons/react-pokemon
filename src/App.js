import React from "react";
import Navbar from "./components/navbar/navbar.component";
import HomePage from "./pages/home/home-page";


function App() {
  return (
    <>
      <Navbar />
      <HomePage />
    </>
  );
}

export default App;
