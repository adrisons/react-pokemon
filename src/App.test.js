import { BrowserRouter as Router } from "react-router-dom";
import renderer from "react-test-renderer";
import App from "./App";

describe("WHEN: PokemonList with no pokemons", () => {
  const component = renderer.create(
    <Router>
      <App />
    </Router>
  );
  const tree = component.toJSON();
  it("THEN: should match snapshot", () => {
    expect(tree).toMatchSnapshot();
  });
});
