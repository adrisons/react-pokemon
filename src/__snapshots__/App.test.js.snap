// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WHEN: PokemonList with no pokemons THEN: should match snapshot 1`] = `
Array [
  <div
    className="navbar"
  >
    <a
      className="logo-container"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="logo"
        className="logo"
        src="pokemon-logo.png"
      />
    </a>
  </div>,
  <div
    className="loading"
  >
    <img
      alt="loading"
      src="pokeball.png"
    />
  </div>,
]
`;
